include/deal.II/grid/grid_generator.h

```diff
@@ -135,6 +135,17 @@ namespace GridGenerator
    * <tt>Triangulation@<2,3@></tt> will be a square in the xy plane with z=0.
    *
    * @note The triangulation passed as argument needs to be empty when calling this function.
+   *
+   * @param tria The Triangulation to create. It needs to be empty upon
```

Comment:
```suggestion
   * @param tria The triangulation to create. It needs to be empty upon
```
Let's be consistent with respect to two lines above here.